.appveyor.yml

```diff
@@ -49,22 +50,21 @@ install:
   - set PATH=%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\scripts;%PATH%;
   - set PYTHONUNBUFFERED=1
   - conda config --set always_yes true
-  - conda update --all
   - conda config --set show_channel_urls yes
   - conda config --prepend channels conda-forge
-  # this is now the downloaded conda...
-  - activate
-  - conda info -a
 
   # For building, use a new environment which only includes the requirements for mpl
   # if conda-forge gets a new pyqt, it might be nice to install it as well to have more backends
   # https://github.com/conda-forge/conda-forge.github.io/issues/157#issuecomment-223536381
   - conda create -q -n test-environment python=%PYTHON_VERSION%
-    freetype=2.6 tk=8.5
-    pip setuptools numpy sphinx tornado
+    pip setuptools wheel tk
```

Comment:
also, conda's python comes with pip and setuptools by default (can be checked with `conda info pip`) -- this should be the case of most pythons anyways (unless split into pieces by a distro packagern basically).

Reply:
I remember I tried to remove them and it did not work, but lets try it again

Reply:
Runs fine